### ISSUE TITLE
response 422 code status

### ISSUE THREAD

**Issue by denniscastro** [mike-north/ember-api-actions#64]
Hi.

Good addon, but what about when response is 422 and ember return and DS.invalidError object?

**Comment by mike-north** [comment]
Please propose a concrete scenario, including what you'd attempt to persist, the response you'd expect to get back, and how you'd expect the addon to behave.

There's some automatic error handling in ember-data that goes with implementing the JSON-API standard, but this is a more general addon that doesn't assume anything about your UI/API contracts

**Comment by malina** [comment]
If i use default ember action save
```
someModel.set('someField', 'someValue');
someModel.save();
```
And if response is 422 than model has errors.
```
someModel.get('errors).someField  // error object with message from server
```

And if i use custom action
```
someModel.customAction({someField: 'someValue'});
```

And if response is 422 than model has  no errors.
```
someModel.get('errors).someField  // is null
```

**Comment by denniscastro** [comment]
Yes.. You need to push the response.errors to Ds.invalidErrors and/or Ds.Error

**Comment by mike-north** [comment]
This is beyond the scope of this addon. ember-api-actions makes no assumptions about the nature of your payload (i.e., whether you're passing a model to your API _at all_), so handling validation errors wouldn't make sense as a default.

What we could do is provide a standard pattern for dealing with errors in the README

**Comment by Exelord** [comment]
@denniscastro to get this behavior works check my package with implemented push to store option.
https://github.com/Exelord/ember-custom-actions/
